### DISCUSSION TITLE
Dedicated Docusaurus actions organization

### DISCUSSION THREAD

**Discussion by henrynewman** [facebook/docusaurus#9001] [Show and tell]
Greetings everyone, I have created a dedicated organization for  GitHub Actions to help deploy Docusaurus. Your inputs, improvements, and contributions are highly welcomed. Available here - [Docusaurus Actions](https://github.com/docuactions) (https://github.com/docuactions)

**Comment by slorber** [comment]
Thanks, looks like a good idea

I think at some point we'll want to provide our own official Docusaurus GH cache action (an official version of https://github.com/docuactions/cache)

For deploying to Netlify I would recommend to just use their UI/CI (like we do) instead of using GitHub Actions

**Comment by andrewnicols** [comment]
@henrynewman, what happened? It appears that docuactions has disappeared?

**Reply by henrynewman** [reply]
Accidentally deleted it whilst updating the Actions, I have contacted Github Support to restore it, awaiting their response.

**Reply by gaby** [reply]
@henrynewman Any updates?

**Reply by henrynewman** [reply]
Not at the moment still waiting for GitHub to respond.

**Comment by henrynewman** [comment]
@gaby @andrewnicols DocuActions has been restored. Need some assistance from the community on updating some more of the actions haven’t been having much time to do it myself.

**Reply by gaby** [reply]
@henrynewman Thanks! 💪
